lib/next_ls/db.ex

```diff
@@ -93,10 +93,10 @@ defmodule NextLS.DB do
     __query__(
       {conn, s.logger},
       ~Q"""
-      INSERT INTO symbols (module, file, type, name, line, 'column', source)
-          VALUES (?, ?, ?, ?, ?, ?, ?);
+      INSERT INTO symbols (module, file, type, name, line, start_column, end_column, source)
+          VALUES (?, ?, ?, ?, ?, ?, ?, ?);
       """,
-      [mod, file, "defmodule", mod, module_line, 1, source]
+      [mod, file, "defmodule", mod, module_line, 1, 1, source]
```

Comment:
I am not sure if it would be better to just not store the `start_column` and `end_column` in this case. The current information is not accurate and it may be more confusing than not having it.

This would also make it easier to query for symbols as we would just return the results in which the column matches or there is no column. Currently we have to discard `defmodule` and `defstruct` manually when querying.

Reply:
So, for defstruct, I believe we can query the reference table to get the start/end column information. i _think_ that since the module is now compiled, all of the references in it should already have been traced.

For defmodule, it seems like only calls to `defmodule` that are inside another module are traced (not sure if that is a bug or an implementation detail), but you can most likely use it for column info for sub moduled, modules, and then if that doesn't exist, just assume its normal and its at column 1 (or 0, i can't remember how we store it)

Reply:
I've just added a new commit that does this and it seems to work fine both for `defstruct` and nested `defmodule` declarations.

For top level `defmodule` declarations we don't have this information and are storing both `start_column` and `end_columns` as 1.

I was thinking that we could do a little bit better as we know that the start column is 11 ("defmodule " has 10 characters) and we can add the module name length to get the end column. Not sure about this though as it seems too "clever" and fragile.